code/obj/machinery/law_racks.dm

```diff
@@ -615,6 +619,51 @@
 		UpdateIcon()
 		UpdateLaws()
 
+	proc/insert_videocard_callback(var/mob/user, var/obj/item/peripheral/videocard/I)
+		var/mob/living/target = null
+		user.u_equip(I)
+		I.set_loc(src)
+		playsound(src, "sound/machines/interdictor_activate.ogg", 70, extrarange=3)
+		src.visible_message("<span class='alert'>[I] emits a loud whirring noise as it connects into the [src]!</span>")
+		SPAWN(7 SECONDS)
+			playsound(src, "sound/machines/engine_highpower.ogg", 80)
+			src.use_power(500)
+			for (var/i in 1 to 10)
+				sleep(0.3 SECONDS)
+				if(src && prob(60))
+					var/obj/mined = new /obj/item/spacecash/buttcoin
+					mined.set_loc(src.loc)
+					target = orange(1,src)[rand(1,8)]
```

Comment:
Pretty sure this could cause an out-of-range exception if there aren't objects in range.

Reply:
should maybe be 
```
var/listl/targetlist = orange(1,src)
target = null
if(length(targetlist))
    target = pick(target)
```

Reply:
switched to get_step, so the rack can still eject the soon-to-be-on-fire card away from itself